{
  "uid" : "2cca9a9f8eb6e1b0",
  "name" : "test_fun_01",
  "fullName" : "test_case#test_fun_01",
  "historyId" : "889aba88ac156d81da8970ed1159bfe9",
  "time" : {
    "start" : 1610630976908,
    "stop" : 1610630976908,
    "duration" : 0
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
  "statusTrace" : "@pytest.mark.usefixtures(\"function_no_return_fixture\")\n    def test_fun_01():\n        print(\"我是第二个测试函数\");\n>       assert fun(1) ==5;\nE       assert 4 == 5\nE        +  where 4 = fun(1)\n\ntest_case.py:54: AssertionError",
  "flaky" : true,
  "beforeStages" : [ {
    "name" : "function_no_return_fixture",
    "time" : {
      "start" : 1610630976907,
      "stop" : 1610630976907,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
    "statusTrace" : "@pytest.mark.usefixtures(\"function_no_return_fixture\")\n    def test_fun_01():\n        print(\"我是第二个测试函数\");\n>       assert fun(1) ==5;\nE       assert 4 == 5\nE        +  where 4 = fun(1)\n\ntest_case.py:54: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "de244c203fd25752",
      "name" : "stdout",
      "source" : "de244c203fd25752.txt",
      "type" : "text/plain",
      "size" : 102
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "function_no_return_fixture::0",
    "time" : {
      "start" : 1610630976910,
      "stop" : 1610630976910,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('function_no_return_fixture')"
  }, {
    "name" : "suite",
    "value" : "test_case"
  }, {
    "name" : "host",
    "value" : "yantianpeng.local"
  }, {
    "name" : "thread",
    "value" : "28074-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_case"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "14c0d808edaedb5d",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630976611,
        "stop" : 1610630976611,
        "duration" : 0
      }
    }, {
      "uid" : "29ac88b909029ad9",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630976611,
        "stop" : 1610630976611,
        "duration" : 0
      }
    }, {
      "uid" : "fde6401d2be25c24",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630976256,
        "stop" : 1610630976256,
        "duration" : 0
      }
    }, {
      "uid" : "9b6279f24788da97",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630976256,
        "stop" : 1610630976256,
        "duration" : 0
      }
    }, {
      "uid" : "d36b139e4dfecaa8",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630976256,
        "stop" : 1610630976256,
        "duration" : 0
      }
    }, {
      "uid" : "49f03f546ea8ad6e",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630975856,
        "stop" : 1610630975857,
        "duration" : 1
      }
    }, {
      "uid" : "e00f6247bcb8a834",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630975856,
        "stop" : 1610630975857,
        "duration" : 1
      }
    }, {
      "uid" : "f3ba5a95a2be270f",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630975856,
        "stop" : 1610630975857,
        "duration" : 1
      }
    }, {
      "uid" : "521ef76910b99674",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 4 == 5\n +  where 4 = fun(1)",
      "time" : {
        "start" : 1610630975856,
        "stop" : 1610630975857,
        "duration" : 1
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "@pytest.mark.usefixtures('function_no_return_fixture')" ]
  },
  "source" : "2cca9a9f8eb6e1b0.json",
  "parameterValues" : [ ]
}